ios-iTunes-search/ios-iTunes-search/SearchResultTableViewCell.swift

```diff
@@ -0,0 +1,22 @@
+//
+//  SearchResultTableViewCell.swift
+//  ios-iTunes-search
+//
+//  Created by Lambda_School_Loaner_268 on 2/11/20.
+//  Copyright © 2020 Lambda. All rights reserved.
+//
+
+import UIKit
+
+class SearchResultTableViewCell: UITableViewCell {
+
+    @IBOutlet weak var titleLabel: UILabel!
+    
+    @IBOutlet weak var artistLabel: UILabel!
+    //MARK: - Methods
+    func updateViews() {
+        titleLabel.text = "title"
+        artistLabel.text = "creator"
+    }
+
+}
```

Comment:
No need for this file since we are not doing a custom cell. Whenever we use Apple's standard cells like Basic, Subtitle or Right Detail we don't need to make a TableViewCell File. 
If you do delete this file you will want to clear the class in the Identity Inspector as well as deleting the outlets on the TableViewController.